src/lib/data/history.ts

```diff
@@ -39,6 +39,7 @@ export async function generateTCX(rideSummary: RideSummaryModel) {
         ...(record.speed && {
           extensions: new TrackPointExtensions({
             Speed: record.speed,
+            Watts: record.instantPower || 0,
```

Comment:
Is it better to do not set it if we don't have it? Like: `record.instantPower && ...({Watts: record.instantPower})`

Reply:
yeah that would be better, didn't know that syntax. How does that actually work?